webapp/packages/plugin-data-viewer/src/ValuePanelPresentation/ImageValue/ImageValuePresentation.tsx

```diff
@@ -154,11 +153,11 @@ export const ImageValuePresentation: TabContainerPanelComponent<IDataValuePanelP
           if (isResultSetFileValue(this.cellValue)) {
             return false;
           }
-          if (isResultSetContentValue(this.cellValue)) {
-            if (this.cellValue.binary) {
-              return this.contentAction.isContentTruncated(this.cellValue);
-            }
+
+          if (this.selectedCell && this.formatAction.isBinary(this.selectedCell)) {
```

Comment:
i`sResultSetBinaryValue(this.cellValue)` - this part broke the feature here. so I went with this check